### ISSUE TITLE
[Protocol overview] Elaborate on overview

### ISSUE THREAD

**Issue by oskarth** [status-im/specs#28]
In https://github.com/status-im/specs/blob/master/x4.md#protocol-overview

    why p2p?
    why on Ethereum stack?
    designed for mobile from the beginning

Write a paragraph or two on this.

**Comment by corpetty** [comment]
As it currently stands, it isn't obvious what doc this would live in.

**Comment by oskarth** [comment]
I think we can ignore this for now, too wordy. Closing
